commons-dlp/src/main/resources/db/changelog/exo-dlp.db.changelog-1.0.0.xml

```diff
@@ -65,5 +65,16 @@
       <column name="ITEM_TITLE" type="VARCHAR(250)"/>
     </addColumn>
   </changeSet>
+
+  <changeSet author="exo-dlp" id="1.0.0-6">
+    <createTable tableName="RESTORED_DLP_ITEMS">
+      <column name="RESTORED_ITEM_ID" type="BIGINT" autoIncrement="${autoIncrement}" startWith="1">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="PK_RESTORED_DLP_ITEMS"/>
+      </column>
+      <column name="RESTORED_ITEM_REFERENCE" type="VARCHAR(50)"/>
+      <column name="RESTORED_DETECTION_DATE" type="TIMESTAMP"/>
+    </createTable>
+    <createSequence sequenceName="SEQ_RESTORED_DLP_ITEMS_ID" startValue="1"/>
```

Comment:
the createSequence should be inside another changetset, applicable only for psql